### PR TITLE
Remove DCGM patch and enforce version from version map

### PR DESCRIPTION
**Pull request by jbkyang-nvi**
Remove DCGM patch and enforce version from version map

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -70,7 +70,8 @@
        '21.06',      # upstream container
        '1.8.0',      # ORT
        '2021.2.200', # ORT OpenVINO
-       '2021.2')     # Standalone OpenVINO
+       '2021.2',     # Standalone OpenVINO
+       '2.2.8')      # DCGM version
```
Comment (CoderHam) [inline build.py:74]: Need to update on L59

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: build.py (hunk 1 above)
Comment (CoderHam) [reply build.py:74]: @dzier / @deadeyegoodwin since we are using a specific version of DCGM maybe we should add this to the release notes?

Comment (deadeyegoodwin) [reply build.py:74]: Good idea. Can you open a ticket as a reminder for this.

Comment (CoderHam) [reply build.py:74]: Done: DLIS-2710

Comment (deadeyegoodwin) [reply build.py:74]: @jbkyang-nvi  while you are in this code can you also combine 58 and 59 to just "Standalone OpenVINO version"

File: build.py [hunk 2]
```diff
@@ -474,16 +475,21 @@ def dali_cmake_args():
 
 
 def install_dcgm_libraries():
+    dcgm_version = ''
+    if FLAGS.version not in TRITON_VERSION_MAP:
+        fail('unable to determine default repo-tag, container version not known for {}'.format(FLAGS.version))
```
Comment (deadeyegoodwin) [inline build.py:480]: "... DCGM version not know for ..."

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

**Review by CoderHam — APPROVED** [review]

**Pull request merged**